### PR TITLE
Install guide: add signposting about chosen operation system and breadcrumb to guide start

### PR DESCRIPTION
**Pull request by vickytnz**
Added:
- breadcrumb at top of pages of operating-model specific steps to link back to start of install guide https://prototype-kit.service-manual.nhs.uk/install/simple . Not added to system choice as this is a switcher page and has a link at the bottom to get back to the install home
- variable on steps to show operation system (windows or mac) set on the child pages with a nunjucks variable (this is used rather than the data from the radio buttons in case someone comes to the pages directly as choosing your operating system is only for triaging to the right pages). System added after steps to be reassurance but less repetitive than being at the beginning (Mac: Step 1 of 8) 

The advanced guide is not changed as it is one level down from the get started guide and a single page so does not need extra breadcrumbs.

### Mac guide
| Before | After |
| -------| ------ |
| Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 1 of 8' |  Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 1 of 8: Mac' |  
| Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 2 of 8' | Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 2 of 8: Mac' |
 | Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 3 of 8' | Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 3 of 8: Mac'|
| Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 4 of 8' |  Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 4 of 8: Mac' |
| Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 5 of 8' | Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 5 of 8: Mac' |
| Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 6 of 8' | Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 6 of 8: Mac' | 
| Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 7 of 8' |  Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 7 of 8: Mac' |
 | Mac install guide page as-is with breadcrumb link to 'Get started' and 'step 8 of 8' | Mac install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 8 of 8: Mac' | 

### WIndows guide

| Before | After |
| -------| ------ |
| Window install guide page as-is with breadcrumb link to 'Get started' and 'step 1 of 8' | Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 1 of 8: Windows' | 
| Window install guide page as-is with breadcrumb link to 'Get started' and 'step 2 of 8' |  Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 2 of 8: Windows' |
| Window install guide page as-is with breadcrumb link to 'Get started' and 'step 3 of 8' | Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 3 of 8: Windows' | 
|  Window install guide page as-is with breadcrumb link to 'Get started' and 'step 4 of 8'  |  Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 4 of 8: Windows' | 
|  Window install guide page as-is with breadcrumb link to 'Get started' and 'step 5 of 8' | Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 5 of 8: Windows' |
| Window install guide page as-is with breadcrumb link to 'Get started' and 'step 6 of 8' | Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 6 of 8: Windows'  |
| Window install guide page as-is with breadcrumb link to 'Get started' and 'step 7 of 8' | Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 7 of 8: Windows' |
| Window install guide page as-is with breadcrumb link to 'Get started' and 'step 8 of 8' | Window install guide page with breadcrumb link to 'Get started' and 'Install guide' and 'step 8 of 8: Windows' |

## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ ] CHANGELOG entry

### CODE REVIEW

File: app/views/install/node.html [hunk 1]
```diff
@@ -23,7 +28,7 @@
     <div class="nhsuk-grid-column-two-thirds">
 
       <h1>
-        <span class="nhsuk-caption-l">Step 2 of 8</span>
+        <span class="nhsuk-caption-l">Step 2 of 8: {{system | default ("[add operating system with 'system']") }}</span>
```
Comment (frankieroberto) [inline app/views/install/node.html:31]: Might be safer to do nothing if we we forget to set `system` on a page?

```suggestion
        <span class="nhsuk-caption-l">Step 2 of 8{% if system %}: {{system }}{% endif %}</span>
```

File: app/views/install/download.html [hunk 2]
```diff
@@ -23,7 +28,7 @@
     <div class="nhsuk-grid-column-two-thirds">
 
       <h1>
-        <span class="nhsuk-caption-l">Step 4 of 8</span>
+        <span class="nhsuk-caption-l">Step 4 of 8: {{system | default ("[add operating system with 'system']") }}</span>
```
Comment (frankieroberto) [inline app/views/install/download.html:31]: Might be safer to do nothing if we we forget to set `system` on a page?

```suggestion
        <span class="nhsuk-caption-l">Step 4 of 8{% if system %}: {{system }}{% endif %}</span>
```

**Review by frankieroberto — APPROVED** [review]
Nice! Minor suggestion but not blocking.

I'd also be tempted to rename it `operatingSystem` for clarity, as `system` is a little vague?

**Comment by vickytnz** [comment]
Changes made to:
- make the variable operatingSystem for legibility
- remove the default text in case it's left out by accident
- changed the code to also work better if empty with the terse but effective if expression `{{[":", operatingSystem ] | join(" ") if operatingSystem }}` 

Ideally i'd like to refactor these templates at some point to make some things more baked into the templates but not a priority.

**Pull request merged**